app/react/Markdown/components/BarChart.js

```diff
@@ -51,21 +65,25 @@ export const BarChartComponent = (props) => {
 
 BarChartComponent.defaultProps = {
   context: 'System',
+  layout: 'horizontal',
+  maxCategories: '0',
```

Comment:
This will certainly break every pie and chart out there.  Should we default to something 'useful'?